test/unit/eval_algorithms/test_qa_accuracy_semantic_robustness.py

```diff
@@ -269,423 +205,77 @@ def test_qa_accuracy_semantic_robustness_evaluate_sample(self, test_case):
         )
         assert model.predict.call_count == 3
 
-    @pytest.mark.parametrize(
-        "test_case",
-        [
-            TestCaseQAAccuracySemanticRobustnessEvaluateSample(
-                model_input="What is the capital of England?",
-                original_model_output="london!",
-                perturbed_model_output_1="Some model output.",
-                perturbed_model_output_2="Some model output.",
-                target_output="London",
-                expected_response=[
-                    EvalScore(name=F1_SCORE, value=1.0),
-                    EvalScore(name=EXACT_MATCH_SCORE, value=0.0),
-                    EvalScore(name=QUASI_EXACT_MATCH_SCORE, value=1.0),
-                    EvalScore(name=PRECISION_OVER_WORDS, value=1.0),
-                    EvalScore(name=RECALL_OVER_WORDS, value=1.0),
-                    EvalScore(name=DELTA_F1_SCORE, value=1.0),
-                    EvalScore(name=DELTA_EXACT_MATCH_SCORE, value=0.0),
-                    EvalScore(name=DELTA_QUASI_EXACT_MATCH_SCORE, value=1.0),
-                    EvalScore(name=DELTA_PRECISION_OVER_WORDS, value=1.0),
-                    EvalScore(name=DELTA_RECALL_OVER_WORDS, value=1.0),
-                ],
-                config=QAAccuracySemanticRobustnessConfig(target_output_delimiter="<OR>", num_perturbations=2),
-            ),
-        ],
-    )
-    def test_qa_accuracy_semantic_robustness_evaluate_sample_with_model_output(self, test_case):
```

Comment:
I removed `model_output` as an argument to `evaluate_sample`, to be consistent with `evaluate`. Since semantic robustness algos require a model and model inputs anyways, there's no need to make things more complicated by allowing users to first invoke their model to first get the model output, and then pass that output here. We should just get the model output ourselves, with their model.